web_text_widget/__openerp__.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2015 OpenERP s.a. (<http://openerp.com>).
+#    Copyright (C) 2015 initOS GmbH & Co. KG (<http://www.initos.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Web text limited widget',
+    'version': '0.1.0',
+    'author': 'initOS GmbH & Co. KG',
+    'category': '',
+    'description': """
+
+* Add new functionality for TextField, but data are limited to
```

Comment:
Probably it applies only to Text fields and not to Char fields.
It would be nice - and at least have it in a roadmap - to also support `limit_chars`on Char fields.